src/plugins/xfconf/README.md

```diff
@@ -13,9 +13,38 @@
 
 This is a storage plugin to mount the xfconf configuration settings.
 
+## Xfconf Terminology
+
+### Property
+
+A property in xfconf is the same as a key in libelektra i.e. it has a name and can hold one or more values.
+
+### Channel
+
+A channel is a type of namespace used in the xfconf library.
+Usually, it is used to separate the properties of different applications which is helpful if different applications rely on a property with the same name but require them to hold different values.
+For example Thunar uses a channel named `thunar`, Xfwm uses a channel named `xfwm4` and so on.
+Keep in mind that channels are only used to separate the properties such as namespaces.
+They are not a security feature i.e. every application has read/write access to every channel.
+
+The list of all channels is stored in the `system:/elektra/modules/xfconf/channels` which is an array of all channel names.
+Channel cannot be explicitly created or removed.
+They only exist in an implicit manner when you pass the `channel=...` argument.
```

Comment:
I don't understand this sentence. Do you mean "plugin configuration" when you say argument? Please describe how channels are used within xfce4, and how they map to Elektra.

Reply:
I hope the commit e282a32bd240e8ee29ac5ede3426e3e422cc9f7a makes it more clear,